### ISSUE TITLE
phpMyAdmin-4.7.0-all-languages.tar.gz contains a git submodule in phpMyAdmin/vendor/codacy/coverage/

### ISSUE THREAD

**Issue by estrabd** [phpmyadmin/phpmyadmin#13201]
### Steps to reproduce
1. download https://files.phpmyadmin.net/phpMyAdmin/4.7.0/phpMyAdmin-4.7.0-all-languages.tar.gz
2. tar zxvf
3. observe phpMyAdmin/vendor/codacy/coverage/.git

### Expected behaviour
I suspect all git repos would be out of this tarball.

### Actual behaviour
Tarball includes a git repo as a submodule

### Server configuration
**Operating system**:
Any

**Comment by Achilles-96** [comment]
Codacy is a require-dev package. This should have been fixed by #13146.

**Comment by estrabd** [comment]
@Achilles-96 - so would be available in the 4.7.0 tar.gz or will I have to wait for 4.7.1. It's affecting some packaging of phpMyAdmin that I am working on for inclusion in cPanel. Thank you.

**Comment by Achilles-96** [comment]
You may have to wait for 4.7.1.
@nijel , I also see satooshi, squizlabs etc. Why are these included in release? Am I missing something here.

**Comment by ibennetch** [comment]
@estrabd Good catch. This seems to be an artifact of how Codacy is installed, but since we won't include Codacy with future versions, it shouldn't be a problem in the future.

@Achilles-96 Unless I misunderstand your question, that's what was fixed with #13146 - some 'require-dev' packages were mistakenly included in the 4.7.0 release, including satooshi/php-coveralls and squizlabs/php_codesniffer. That's what is fixed with 6f9a50f2e70b48c8e32f1d33538876218ee25093.

We remove .gitignore files from all subdirectories, but seem to leave .gitattributes files scattered around the vendor tree, which I think can be removed at release time. I've opened pull request #13205 to address this.
